Exercise_1.py

```diff
@@ -1,24 +1,73 @@
+
+
+#08:17
+
 class myStack:
   #Please read sample.java file before starting.
   #Kindly include Time and Space complexity at top of each file
+     
      def __init__(self):
-         
+      #  An instance variable of type list is used to implement stack
+       self.stack=[]
+             
+     #  Its a simple method to get length of list variable
      def isEmpty(self):
+       return len(self.stack)==0
          
+    #  This will append an element into list., In python list is an ArrayList 
+    # where once its pre-initialized space is full it will allocate current size of space 
      def push(self, item):
-         
+       self.stack.append(item)
+
+     # pop on Python list  can pop element at top of a list
      def pop(self):
-        
-        
+       if not self.isEmpty():
+         return self.stack.pop()
+
+    # This is implemented by returning last element from list using -1 as index
      def peek(self):
-        
+       if not self.isEmpty():
+        return self.stack[-1]
+
+    #  Returns current length of list
      def size(self):
-         
+       return len(self.stack)
+    
+    # Returns shallow copy of list
      def show(self):
-         
+       return self.stack[:]
 
 s = myStack()
 s.push('1')
 s.push('2')
 print(s.pop())
 print(s.show())
+print(s.isEmpty())
+print(s.peek())
+
+'''
+Time Complexity :
+  isEmpty : O(1)
+  push    : O(N)
```

Comment:
Please check the `push`  operation time  and space complexity.

Reply:
Updated Time and space complexity